keyboards/keebio/bdn9/keymaps/default/keymap.c

```diff
@@ -16,15 +16,26 @@
 #include QMK_KEYBOARD_H
 
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /*
+        | Knob 1: Vol Dn/Up |      | Knob 2: Page Dn/Up |
+        | Press: Mute       | Home | Press: Play/Pause  |
+        | Hold: Layer 2     | Up   | RGB Mode           |
+        | Left              | Down | Right              |
+     */
     [0] = LAYOUT(
-        KC_MUTE, KC_HOME,   KC_MPLY, \
-        MO(1),   KC_UP,   RGB_MOD, \
-        KC_LEFT, KC_DOWN, KC_RGHT \
+        KC_MUTE, KC_HOME, KC_MPLY,
+        MO(1)  , KC_UP  , RGB_MOD,
+        KC_LEFT, KC_DOWN, KC_RGHT
     ),
+    /*
+        | RESET          | N/A  | Media Stop |
```

Comment:
Only correction here is that the middle key changes the brightness of the in-switch LEDs.